content/en/docs/reference/glossary/multicluster-mesh.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Multicluster Mesh
+---
+A [service mesh](#service-mesh) composed of services running within more than one underlying cluster.
+All clusters in a multicluster mesh are under the same administrative control and share the same service definitions.
+A service named `foo` in namespace `ns1` of cluster 1 represents the same service as `foo` in `ns1` of cluster 2.
```

Comment:
Should be part of the "service mesh" glossary entry also.

Reply:
This is specifically talking explaining it in terms of two clusters, cluster 1 and cluster 2. Service mesh may or may not have more than one cluster.